honeybadgermpc/utils.py

```diff
@@ -143,7 +143,7 @@ def type_check(*types, **kwtypes):
     return static_type_check(object, *types, **kwtypes)
 
 
-@static_type_check(str, 'callable(send)')
+@static_type_check(object, 'callable(send)')
```

Comment:
lets stick to `str` only? To make things work, just change line 76 in `butterfly_network.py` from `0` to `"0"`.

Reply:
Sure thing